src/jesse.app.src

```diff
@@ -11,8 +11,6 @@
                    , public_key
                    , ssl
                    , inets
-                   , jsx
```

Comment:
nit: one can actually add .app.src.script, see https://rebar3.org/docs/configuration/config_script

Reply:
Yeah, but we don't actually need jsx here anyways, this list is for apps to be started, jsx is not an app but only code to be loaded, that rebar will put in the right path for the BEAM to load the code automatically

Reply:
it's also for `rebar3 release` to find the apps to be copied.

Reply:
Absolutely very good point I don't know how could I forget! Fixed! :)